values.yaml

```diff
@@ -69,6 +75,9 @@ server:
   # Enables a headless service to be used by the Vault Statefulset
   service:
     enabled: true
+    # clusterIP can be used to disable a cluster IP address from being 
+    # attached to the Vault service if set to "None". 
```

Comment:
Can we re-word this to be clear that the default is to in fact have a cluster IP address attached? The default scenario has to be derived from this sentence, I would prefer something like "by default vault has a cluster ip , set to `None` to disable" and maybe a small bit on why one would disable that

Reply:
Done in https://github.com/hashicorp/vault-helm/pull/10/commits/e8460cd120ae0c5017adfcc80cc4aeb689e99946, thanks!